crates/polars/tests/it/io/csv.rs

```diff
@@ -1389,3 +1391,16 @@ fn test_read_io_reader() {
     let expected = CsvReader::new(file).finish().unwrap();
     assert!(df.equals(&expected))
 }
+
+#[test]
+#[cfg(any(feature = "decompress", feature = "decompress-fast"))]
+fn test_read_compressed() {
+    const COMPRESSED_CSV: &str = "../../examples/datasets/compressed.csv.gz";
```

Comment:
Can we create the compressed file on the fly (in-memory). That saves adding a file to the repo.

Reply:
Current `compressed.csv.gz` is randomly generated and the `n_row` which yields to `ComputeError` is found manually. I'd leave `compressed.csv.gz` as-is since, or remove `test_read_compressed` completely since it mostly here for illustrating the initial issue.

Reply:
Maybe we can create a in memory random file that illustrates this. Otherwise removing the test and file is also fine. :+1:

Reply:
A `test_read_compressed` and a correspondent csv.gz file are removed 👍